IgorMinar/ng-detector
# Add support for Angular 2

This PR adds support for detecting Angular 2 apps with ng-detector.
## Description

When a user visits an Angular 2 app, this extension now adds:
1. A title that says "Super-powered by Angular 2"
2. An blue Angular icon

Regarding item 2, my thinking here is that it would be easier and quicker to see a differently colored icon and recognize an Angular 2 app instead of relying on the title alone. However, this should be consistent with branding for Angular 2, so let me know what your thoughts are here
## To test
1. **Install the extension.** Go to chrome://extensions/ and install the extension
2. **Visit an Angular 1 app.** Go to the [AngularJS.org page](https://angularjs.org/). You should see a red icon with the title "Super-powered by Angular 1"
3. **Visit an Angular 2 app.** Go to [NG2 Dribbble](http://zamamohammed.github.io/ng2-dribbble/#/snaps). You should see a blue icon with the title "Super-powered by Angular 2"
